core-blocks/latest-comments/style.scss

```diff
@@ -0,0 +1,59 @@
+.wp-block-latest-comments {
+	padding-left: 2.5em;
+
+	&__comment-timestamp {
+		color: $dark-gray-100;
+		font-size: 12px;
+		padding: 0 2px;
+	}
+
+	li {
+		line-height: 1.1;
+	}
+
+	&.has-avatars > li {
+		min-height: 36px;
+		list-style: none;
+
+		.comment-data {
+			margin-left: 52px;
+		}
+	}
+
+	&__comment-timestamp {
+		display: block;
+		padding-top: 8px;
+	}
+
+	.wp-block-latest-comments__comment-excerpt p {
+		font-size: 14px;
+		line-height: 1.8;
+		margin: 5px 0 20px;
+	}
+
+	.recentcomments {
+		line-height: 2.2;
+	}
+
+	&.has-timestamps,
+	&.has-excerpts {
+		.recentcomments {
+			line-height: 1.5;
+		}
+	}
+}
+
+.wp-block-latest-comments > li {
```

Comment:
We have styles for the same element in line 10, they should be consolidated there.